### DISCUSSION TITLE
Customize sidebar/title icons

### DISCUSSION THREAD

**Discussion by Nadine2016** [colinwilson/lotusdocs#90] [Q&A]
Is there a way to customize the sidebar and title icons that you enable with `sidebarIcons = true` and `titleIcon = true` so that I can bring my own icons?

**Comment by colinwilson** [comment]
Not at the moment I'm afraid. Just the [Material Symbol Icons](https://fonts.google.com/icons?icon.set=Material+Symbols).

Is it a online particular icon set you're looking at? Or your own custom designed icons?

**Reply by Nadine2016** [reply]
yeah it would be our own custom icons that we would like to add as those icons are used in our UI and other components as well.

**Reply by colinwilson** [reply]
I'm afraid in that case, maintaining your own fork of the project might be the only way to do it.

To use your own icon font requires changes to the theme's HTML and CSS in several places.

**Reply by LuigiColantuono** [reply]
> I'm afraid in that case, maintaining your own fork of the project might be the only way to do it.
> 
> To use your own icon font requires changes to the theme's HTML and CSS in several places.

Most likely you are no longer maintaining this project, but could you explain to me how I could integrate font-awesome?
Because google material is very limited.
I have the desktop and web version, and I have access to practically all the icons in the world, I found some guides that allow me to use FA but in none of them is it explained how to use it in the title (in the icon field).
Lotusdocs from what I have seen manages GM from scss.
I hope you can give me some advice.
Thanks!
